.github/actions/vtk_commit_sha

```diff
@@ -1 +1 @@
-7f5d4608abeaccee3955a3095e0b2d0abf3ed112
+63a4642a4274edb24b0e7ae8b73fd19b3f4aaa33
```

Comment:
which change do you need ?

Reply:
There was an issue with `RenderingWebGPU` module but that's fixed in master. Not sure about which commit fixed it, I'll take a look.

Reply:
Nop cannot find it sorry. Anyway I'll need a new bump when Jaswant MR is merged anyway so it's not really important.

Reply:
no worries